server_py/flatgov/common/crec_data.py

```diff
@@ -0,0 +1,43 @@
+import json
+from bills.models import CommitteeDocument
+from django.conf import settings
+from . import constants
+import re
+
+def get_congress_number(date):
```

Comment:
This calculation makes it seem like 117 is a special number, rather than the current congress. A more general calculation for this is:

`CONGRESS = math.ceil((year - 1788) / 2)`